game/core/src/me/lihq/game/screen/NavigationScreen.java

```diff
@@ -333,11 +333,21 @@ public void updateTiledMapRenderer()
         this.currentNPCS = game.getNPCS(game.player.getRoom());
     }
 
+    public boolean getChangeMap()
+    {
+        return(changeMap);
```

Comment:
The main things that should have tests, are all the objects in the model package and all the people in the game. And if possible the elements of the UI, although again they are hard to test